feat: Add import page

drazik opened the pull request:
It is hidden behind the `import-page` flag for now. It is lacking test (I'm writing some right now) and a real design/wording, but at least the functionality is OK.

When the `import-page` flag is enabled, you have access to a « import my credentials from another manager » link on installed and connected steps that redirects to the `/installation/import` URL:

![image](https://user-images.githubusercontent.com/1606068/75542937-115dc500-5a21-11ea-8f8e-62902c7e8e41.png)

On `/installation/import`, we have something that mimics the import feature from vault.bitwarden.com:

![image](https://user-images.githubusercontent.com/1606068/75542966-20447780-5a21-11ea-900b-8869415e0f55.png)

When the user have selected a format and a file, we ask him to unlock his vault (using VaultUnlocker from cozy-keys-lib, like in Harvest):
![image](https://user-images.githubusercontent.com/1606068/75543008-36523800-5a21-11ea-8799-b5dee69d095f.png)


When the import process is ongoing, we show a spinner:
![image](https://user-images.githubusercontent.com/1606068/75543152-7ca79700-5a21-11ea-9371-ca39c2e83025.png)


When the import is successful, we show a success message:
![image](https://user-images.githubusercontent.com/1606068/75543086-5c77d800-5a21-11ea-8bed-3e7dde5d724c.png)


When the import is errored, we show an error message:
![image](https://user-images.githubusercontent.com/1606068/75543302-c85a4080-5a21-11ea-8a6b-7e96279b7ece.png)

As I said, don't pay too much attention to wording and design, they are mine and will certainly need little updates.

drazik committed:
feat: Add import page

It is hidden behind the `import-page` flag for now

y-lohse commented on src/locales/en.json:
```diff
@@ -72,14 +72,41 @@
+      },
+      "imported": {
+        "title": "Credentials imported",
+        "content": "You can now find your credentials in your cozy vault."
```
```suggestion
        "content": "You can now find your credentials in your Cozy vault."
```

y-lohse commented on src/components/ImportPage/index.jsx:
```diff
@@ -0,0 +1,135 @@
+  const [importStatus, setImportStatus] = useState('waiting')
+  const fileInput = useRef(null)
+
+  const importFile = async () => {
```
Nit: we could use `useCallback` for these handlers.

y-lohse reviewed (approved):
LGTM

y-lohse commented:
I haven't found out why the test is failing yet (`https://travis-ci.com/cozy/cozy-passwords/builds/151502590#L234`). It comes from `withLocales` in I18n, I suspect we try to HOC a component that has no display name for some reason. So it might be a bug in cozy-ui and since this PR is urgent, I wonder if we should deactivate the test until the release?

ptbrowne commented on src/components/ImportPage/helpers.js:
```diff
@@ -0,0 +1,38 @@
+        if (pre != null) {
+          resolve(pre.textContent)
+          return
+        }
```
Nit: The above lines should be in a parseLastpass function, this would be clearer and would remove the need of part the docstring. Since it's taken copied from another file, I would understand not to change it though (if in the future we need to copy it again).

y-lohse committed:
test: Remove cozy client mock

y-lohse committed:
Update src/locales/en.json

ptbrowne committed:
chore: Put flag on window

ptbrowne committed:
fix: Add cipher permission

ptbrowne committed:
feat: Update cozy-keys-lib

- No more Hooks errors due to duplicate react version
- Ability to import from Dashlane
